Tests/AssistantV1Tests/AssistantTests.swift

```diff
@@ -97,13 +97,12 @@ class AssistantTests: XCTestCase {
 
     /** Instantiate Assistant. */
     func instantiateAssistant() {
-        let version = "2018-10-10"
         if let apiKey = WatsonCredentials.AssistantAPIKey {
-            assistant = Assistant(version: version, apiKey: apiKey)
+            assistant = Assistant(version: currentDate, apiKey: apiKey)
```

Comment:
I have reservations about this change.  I think 1) it might suggest we think this is good practice -- it is not, and 2) it makes the tests fragile, because behavior change governed by the version date might trigger a failure.